CHANGE_LIMITATIONS.md

```diff
@@ -0,0 +1,21 @@
+# What changes we can't accept
+
+We wish we could accept all changes, especially changes that make the organization or readability of the project better. Unfortunately we can't, due to technical limitations 😢.
```

Comment:
"While we wish we could accept all changes (especially changes that make the organization or readability of the project better), we can't due to technical limitations."

Reply:
💯 I like this wording way better